packages/sd-jwt-vc/src/sd-jwt-vc-status-reference.ts

```diff
@@ -0,0 +1,9 @@
+export interface SDJWTVCStatusReference {
+  // REQUIRED. implenentation according to https://www.ietf.org/archive/id/draft-ietf-oauth-status-list-02.html
```

Comment:
Should we use zod or something to validate the structure of this?

Reply:
I also would not be opposed to introduce zod into this library and heavily rely on it. It offers some very easy to use validation and parsing which should help a lot in general.

Reply:
I opened #227 for this